plugins/hypervisors/ovm3/pom.xml

```diff
@@ -27,11 +27,31 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
   <dependencies>
-<dependency>
-    <groupId>org.apache.xmlrpc</groupId>
-    <artifactId>xmlrpc-client</artifactId>
-    <version>3.1.3</version>
-</dependency>
+    <dependency>
+	<groupId>xml-apis</groupId>
+	<artifactId>xml-apis</artifactId>
+	<version>1.4.01</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.xmlrpc</groupId>
+      <artifactId>xmlrpc-client</artifactId>
+      <version>3.1.3</version>
```

Comment:
We can and should, i just kept this one the way it was, version was already specified in the version currently in master, will update